### PR TITLE
HAL_ChibiOS: added f103-QiotekPeriph

### PR DESCRIPTION
**Pull request by QioTek**
HAL_ChibiOS: added f103-QiotekPeriph

### CODE REVIEW

File: libraries/AP_HAL_ChibiOS/hwdef/f103-QiotekPeriph/hwdef.dat [hunk 1]
```diff
@@ -0,0 +1,24 @@
+include ../f103-periph/hwdef.dat
+
+define CAN_APP_NODE_NAME "org.ardupilot.f103Qiotek_Periph"
+
+# use DNA
+define HAL_CAN_DEFAULT_NODE_ID 0
+
+undef  HAL_COMPASS_MAX_SENSORS 1
+define define HAL_COMPASS_MAX_SENSORS 2
```
Comment (khancyr) [inline libraries/AP_HAL_ChibiOS/hwdef/f103-QiotekPeriph/hwdef.dat:9]: Define define? 
I don't think you the undef line too, but let's wait for someone more qualified to validate

**Review by tridge — APPROVED** [review]

**Comment by tridge** [comment]
looks good, but I do suggest you commit the bootloader in Tools/bootloaders, and add it to build_binaries.py as well

File: libraries/AP_HAL_ChibiOS/hwdef/f103-QiotekPeriph/hwdef.dat (hunk 1 above)
Comment (tridge) [reply libraries/AP_HAL_ChibiOS/hwdef/f103-QiotekPeriph/hwdef.dat:9]: good catch Pierre, I didn't even notice!

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/AP_HAL_ChibiOS/hwdef/f103-QiotekPeriph/hwdef.dat (hunk 1 above)
Comment (QioTek) [reply libraries/AP_HAL_ChibiOS/hwdef/f103-QiotekPeriph/hwdef.dat]: good catch Pierre @khancyr

**Review by tridge — APPROVED** [review]

**Pull request merged**